### DISCUSSION TITLE
Reopen https://github.com/tiangolo/fastapi/issues/4041

### DISCUSSION THREAD

**Discussion by haf** [fastapi/fastapi#8479] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!
```

### Description

Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!

### Wanted Solution

Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!

### Wanted Code

```python
Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!
```

### Alternatives

Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!

### Operating System

macOS

### Operating System Details

Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!

### FastAPI Version

Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!

### Python Version

Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!

### Additional Context

Hi, https://github.com/tiangolo/fastapi/issues/4041 needs reopening!

**Comment by acidjunk** [comment]
Did you already search in Google "#4041 needs reopening"?

**Comment by haf** [comment]
Oh yes! :D 👯 This issue now ranks #1 on google for it. It's just that that issue has been closed for a long while but has attracted a lot of repros in the mean while and there seems to be no way to notify a maintainer to reopen it (and I don't think it's right to e-mail them, so this was a little bit less intrusive)
